### ISSUE TITLE
Live updating on zoom mouse for all platforms

### ISSUE THREAD

**Issue by AndreasArvidsson** [talonvoice/talon#504]
It's problematic with animated webpage menus when the zoomed in view doesn't show that the menu has collapsed and you end up clicking on something in the background instead.

**Comment by lunixbochs** [comment]
Are you asking for both Linux and Windows? Do you know any way to take screenshots behind the active window on those platforms?

**Comment by AndreasArvidsson** [comment]
Yes I have use of both, but I would be happy to start with windows. Not on top of my head. I know that there are magnifier applications for at least windows that has this kind of feature so it should be possible.
